Add DDR experiment spec (#300)
TheTorProject/ooni-spec: data-formats/df-002-dnst.md

- `serial_number` (`int`; only for SOA answers): version number
of the original copy of the zone.

- `svcb`: (only for SCVB answers): the SVCB record, see below.

- `ttl` (`int`; nullable): the TTL if known, otherwise `null`.

### SVCB answer

Answers with the SVCB record contain the `svcb` key that is an object like this:

```Javascript
{
    "priority": 1,
    "target_name": "dns.quad9.net.",
    "params": {
        "alpn": "h2",
        "dohpath": "/dns-query{?dns}",
        "ipv4hint": "9.9.9.9,149.112.112.112",
        "ipv6hint": "2620:fe::fe",
        "port": "443"
    }
}
```
The svcb object follows the SVCB specifation in RFC9460.

- `priority`: (`string`) The SvcPriority
  
- `target_name`: (`string`) The TargetName as it appears in the record.

> - `params`: The SvcParams, as an object with string values.

Reviewer: ```suggestion
- `params`: (`map[string]string`) The SvcParams, as an map from string to string.
```

> - `params`: (`map[string]string`) The SvcParams, as an map from string to string.

Reviewer 2: ```suggestion
- `params`: (`map[string]string`) The SvcParams, as a map from string to string.
```
